gensim/models/fasttext.py

```diff
@@ -27,10 +30,87 @@
 
 .. sourcecode:: pycon
 
-    >>> from gensim.test.utils import common_texts
-    >>> from gensim.models import FastText
+    >>> # from gensim.models import FastText  # FIXME: why does Sphinx dislike this import?
+    >>> from gensim.test.utils import common_texts  # some example sentences
+    >>> print(common_texts[0])
```

Comment:
good idea to add an empty line (`>>> `) between imports and actual commands, here and everywhere

Reply:
Done